2019-12-08 16:57:22,034 DEBUG   MainThread:19328 [wandb_config.py:_load_defaults():87] no defaults not found in config-defaults.yaml
2019-12-08 16:57:22,074 DEBUG   MainThread:19328 [util.py:is_cygwin_git():318] Failed checking if running in CYGWIN due to: FileNotFoundError(2, 'The system cannot find the file specified', None, 2, None)
2019-12-08 16:57:22,079 DEBUG   MainThread:19328 [cmd.py:execute():724] Popen(['git', 'cat-file', '--batch-check'], cwd=C:\Users\joon0\dev\hierarchical_qmix, universal_newlines=False, shell=None, istream=<valid stream>)
2019-12-08 16:57:22,150 DEBUG   MainThread:19328 [cmd.py:execute():724] Popen(['git', 'rev-parse', '--show-toplevel'], cwd=C:\Users\joon0\dev\hierarchical_qmix, universal_newlines=False, shell=None, istream=None)
2019-12-08 16:57:22,224 DEBUG   MainThread:19328 [cmd.py:execute():724] Popen(['git', 'status', '--porcelain', '--untracked-files'], cwd=C:\Users\joon0\dev\hierarchical_qmix, universal_newlines=False, shell=None, istream=None)
2019-12-08 16:57:22,321 DEBUG   MainThread:19328 [run_manager.py:__init__():535] Initialized sync for qmix3/tjhpxz1y
2019-12-08 16:57:22,327 INFO    MainThread:19328 [run_manager.py:wrap_existing_process():1115] wrapping existing process 6252
2019-12-08 16:57:22,331 DEBUG   MainThread:19328 [connectionpool.py:_new_conn():959] Starting new HTTPS connection (1): pypi.org:443
2019-12-08 16:57:22,917 DEBUG   MainThread:19328 [connectionpool.py:_make_request():437] https://pypi.org:443 "GET /pypi/wandb/json HTTP/1.1" 200 38505
2019-12-08 16:57:23,041 INFO    Thread-7  :19328 [run_manager.py:_on_file_created():671] file/dir created: C:\Users\joon0\dev\hierarchical_qmix\src\test\wandb\run-20191208_075720-tjhpxz1y\output.log
2019-12-08 16:57:23,201 INFO    MainThread:19328 [run_manager.py:init_run():912] system metrics and metadata threads started
2019-12-08 16:57:23,201 INFO    MainThread:19328 [run_manager.py:init_run():946] upserting run before process can begin, waiting at most 10 seconds
2019-12-08 16:57:23,211 DEBUG   Thread-20 :19328 [connectionpool.py:_new_conn():959] Starting new HTTPS connection (1): api.wandb.ai:443
2019-12-08 16:57:23,597 DEBUG   Thread-20 :19328 [connectionpool.py:_make_request():437] https://api.wandb.ai:443 "POST /graphql HTTP/1.1" 200 492
2019-12-08 16:57:23,603 INFO    Thread-20 :19328 [run_manager.py:_upsert_run():1019] saving patches
2019-12-08 16:57:23,603 DEBUG   Thread-20 :19328 [cmd.py:execute():724] Popen(['git', 'rev-parse', '--show-toplevel'], cwd=C:\Users\joon0\dev\hierarchical_qmix, universal_newlines=False, shell=None, istream=None)
2019-12-08 16:57:23,679 DEBUG   Thread-20 :19328 [cmd.py:execute():724] Popen(['git', 'diff', '--cached', '--abbrev=40', '--full-index', '--raw'], cwd=C:\Users\joon0\dev\hierarchical_qmix, universal_newlines=False, shell=None, istream=None)
2019-12-08 16:57:23,764 DEBUG   Thread-20 :19328 [cmd.py:execute():724] Popen(['git', 'version'], cwd=C:\Users\joon0\dev\hierarchical_qmix, universal_newlines=False, shell=None, istream=None)
